week1/Afshin Week1.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # ex1
+
+name = input ("Enter your name :")
+surname = input ("Enter your surname :")
+string_print = f"Your name is {name} and your family is {surname }."
+print (string_print)
+
+
+# ex2
+i = 1
+x = ""
+while i < 6:
```

Comment:
You should only use while loops if you really need to, it's better practice to use for loops in python.
A for loop that counts from 0 to 4 looks like this:
```python
for i in range(5):
    print(i)
```